packages/features/users/components/UserTable/EditSheet/EditUserForm.tsx

```diff
@@ -189,15 +212,160 @@ export function EditForm({
               }}
             />
           </div>
-          <div>
+          <div className="mb-4">
             <Label>{t("timezone")}</Label>
             <TimezoneSelect value={watchTimezone ?? "America/Los_Angeles"} />
           </div>
+          <AttributesList selectedUserId={selectedUser?.id} />
         </SheetBody>
         <SheetFooter>
-          <SheetFooterControls />
+          <Button
+            color="secondary"
+            type="button"
+            className="justify-center md:w-1/5"
+            onClick={() => {
+              setEditMode(false);
+            }}>
+            {t("cancel")}
+          </Button>
+
+          <Button type="submit" className="w-full justify-center">
+            {t("update")}
+          </Button>
         </SheetFooter>
       </Form>
     </>
   );
 }
+
+type AttributeType = z.infer<typeof attributeSchema>;
+
+type DefaultValueType = {
+  [key: `attributes.${number}`]: AttributeType;
+};
+
+function AttributesList(props: { selectedUserId: number }) {
+  const { data: usersAttributes, isPending: usersAttributesPending } =
+    trpc.viewer.attributes.getByUserId.useQuery({
+      userId: props.selectedUserId,
+    });
+  const { data: attributes, isPending: attributesPending } = trpc.viewer.attributes.list.useQuery();
+  const enabledAttributes = attributes?.filter((attr) => attr.enabled);
+
+  const { t } = useLocale();
+  const { control, watch, getFieldState, setValue } = useFormContext();
+
+  // Watch the 'attributes' field from the form context
+  const formAttributes = watch("attributes") as AttributeType[];
+
+  const getOptionsByAttributeId = (attributeId: string) => {
+    const attribute = attributes?.find((attr) => attr.id === attributeId);
+    return attribute
+      ? attribute.options.map((option) => ({
+          value: option.id,
+          label: option.value,
+        }))
+      : [];
+  };
+
+  const defaultValues = useMemo<DefaultValueType>(() => {
+    if (!usersAttributes || usersAttributesPending || !enabledAttributes) return {};
+
+    return enabledAttributes.reduce<DefaultValueType>((acc, enabledAttr, index) => {
+      const attr = usersAttributes.find((attr) => attr.id === enabledAttr.id);
+      const key = `attributes.${index}` as const;
+
+      if (!attr) {
+        acc[key] = { id: enabledAttr.id, value: "", options: [] };
+      } else if (attr.type === "MULTI_SELECT") {
+        acc[key] = {
+          id: attr.id,
+          options: attr.options.map((option) => ({ label: option.value, value: option.id })),
+        };
+      } else if (attr.type === "SINGLE_SELECT") {
+        acc[key] = {
+          id: attr.id,
+          options: [{ label: attr.options[0]?.value, value: attr.options[0]?.id }],
+        };
+      } else {
+        acc[key] = {
+          id: attr.id,
+          value: attr.options[0]?.value || "",
+        };
+      }
+      return acc;
+    }, {});
+  }, [usersAttributes, usersAttributesPending, enabledAttributes]);
```

Comment:
Merges users attributes from seperate endpoint with all org attributes to create default values